README.md

```diff
@@ -34,6 +34,7 @@ Producer.
 - A JMS Broker (Example: [Apache ActiveMQ](http://activemq.apache.org/getting-started.html))
   * After installing the JMS broker, copy its .jar files into the `<BALLERINA_HOME>/bre/lib` folder
     * For ActiveMQ 5.12.0: Copy `activemq-client-5.12.0.jar` and `geronimo-j2ee-management_1.1_spec-1.0.1.jar`
+    * Later versions of ActiveMQ (eg:- ActiveMQ 5.14.0) may additionally require copying `hawtbuf-1.11.jar`
```

Comment:
Use 'e.g.,' instead of 'eg:-'. 

'e.g.' is a Latin abbreviation that stands for 'exempli gratia' and specifically uses the dots that way. 'i.e.' is similar. Apply elsewhere and in other PRs wherever applicable.

Reply:
Thank you! Fixed with 5ba58e6